src/pages/oracles/index.tsx

```diff
@@ -22,12 +25,18 @@ export default function PricesPage (props: InferGetServerSidePropsType<typeof ge
   const [typeSelection, setTypeCurrentSelection] = useState<string>('All')
   const [availabilitySelection, setAvailabilitySelection] = useState<boolean>(true)
   const tickers: PriceFeedProps[] = []
+  const collateralAndLoanTokensSet: Set<string> = new Set<string>()
```

Comment:
```suggestion
  const collateralAndLoanTokensSet: Set<string> = new Set<string>(props.collateralAndLoanTokens)
```

Not sure if this works